app/src/components/market/sections/market_verify/option/dxdao_curation.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC } from 'react'
+import styled from 'styled-components'
+
+import { IconDxDao } from '../../../../common/icons'
+import Tick from '../img/tick.svg'
+import {
+  CurationCenterColumn,
+  CurationLeftColumn,
+  CurationLogoWrapper,
+  CurationOption,
+  CurationOptionDetails,
+  CurationRadioTick,
+  CurationRadioWrapper,
+  CurationRightColumn,
+  CurationRow,
+  CurationSubRow,
+} from '../market_verify'
+
+const Bold = styled.b`
```

Comment:
Should this be `styled.p`?

Reply:
No, <p> would add a new line with a margin. This is really just supposed to make a text section bold.